src/lib/net/SecureSocket.cpp

```diff
@@ -695,37 +700,34 @@ SecureSocket::verifyCertFingerprint()
     formatFingerprint(fingerprint);
     LOG((CLOG_NOTE "server fingerprint: %s", fingerprint.c_str()));
 
-    std::string trustedServersFilename;
-    trustedServersFilename = barrier::string::sprintf(
-        "%s/%s/%s",
-        DataDirectories::profile().c_str(),
-        kFingerprintDirName,
-        kFingerprintTrustedServersFilename);
+    std::string trustedServersFilename = PathUtilities::concat(DataDirectories::profile(),
+        barrier::string::sprintf("%s%c%s",
```

Comment:
I think we should use PathUtilities::concat here too. Is there any reason not to do that?

Reply:
Just that it is not necessary to perform the searching (for possible removal) of path separators in both ends for every two parts concat'ed together, when I know for sure if the "leftmost" part has a path separator or not, and I am appending n number of simple path segments after it. Would be better if PathUtilites had some flexibility around such cases, will look into it.

Reply:
The chance of bugs outweights the very small performance overhead. If this issue was fixed I could merge the PR.